Add database-manager doc to Getting Started with QGIS

HannahJD opened the pull request:
Add database manager doc and associated images/gifs to Getting Started with QGIS.

wburt commented:
@HannahJD How do you propose linking this with the other documentation? Also there is this --> [adding oracle data with database manager](https://github.com/bcgov/gis-pantry/blob/master/docs/getting-started-with-QGIS/doc/working-with-data-in-QGIS.md#adding-oracle-data-via-db-manager)

HannahJD commented:
Updated TOC to include new doc and added a reference to the DB manager doc within [adding oracle data to QGIS](https://github.com/bcgov/gis-pantry/blob/master/docs/getting-started-with-QGIS/doc/working-with-data-in-QGIS.md#adding-oracle-data-via-db-manager).

wburt commented:
Thanks!
